python/catkin/find_in_workspaces.py

```diff
@@ -75,6 +75,27 @@ def _get_valid_search_dirs(search_dirs, project):
     return search_dirs
 
 
+def collect_path_data_in_workspaces(workspaces=None,
```

Comment:
This function does not provide any new information over the existing catkin API. The algorithm calling the other existing API is trivial. Please move this function to ros/ros_comm#676 where it is being used.